apps/prairielearn/elements/pl-big-o-input/big_o_utils.py

```diff
@@ -152,3 +161,10 @@ def grade_omega_expression(
         return (0.5, CONSTANT_FACTORS_FEEDBACK)
     except TypeError:
         return (0.0, TYPE_ERROR_FEEDBACK)
+
+
+def grade_blank_expression(a_true: str, a_sub: str) -> tuple[int, str]:
```

Comment:
```suggestion
def grade_blank_expression(a_true: str, a_sub: str) -> tuple[float, str]:
```
For consistency with the functions this is called in.